fdbclient/include/fdbclient/BlobTLSConfig.h

```diff
@@ -0,0 +1,37 @@
+/*
+ * BlobTLSConfig.h
+ *
+ * This source file is part of the FoundationDB open source project
+ *
+ * Copyright 2013-2024 Apple Inc. and the FoundationDB project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef FDBCLIENT_BLOBTLSCONFIG_H
+#define FDBCLIENT_BLOBTLSCONFIG_H
+
+// TLS and blob credential setup.
+// Copied from fdbbackup/BackupTLSConfig.* and renamed BlobTLSConfig.
```

Comment:
Maybe we can dedup the code, i.e., removing `fdbbackup/BackupTLSConfig.*` and consolidate with this one.

Reply:
Let me try it!